ci-operator/config/red-hat-data-services/openvino_model_server/red-hat-data-services-openvino_model_server-v2022.3-release.yaml

```diff
@@ -0,0 +1,216 @@
+      - name: CMAKE_BUILD_TYPE
+        value: Release
+      - name: minitrace_flags
+        value: ""
+      - name: PROJECT_NAME
+        value: OpenVINO Model Server
+      - name: PROJECT_VERSION
+        value: 2022.3.27eb5939 # This is from git rev-parse --short HEAD for tracking, can we generate this dynamically? Or is it a Dockerfile change?
+      - name: BASE_IMAGE
+        value: docker.io/nvidia/cuda:11.8.0-runtime-ubi8 # nvcr.io has one too, there's also the base_image stuff above, should it be there?
+      - name: NVIDIA
+        value: "1"
+      - name: ov_contrib_branch
+        value: "releases/2022/3"
+      - name: JOBS
+        value: "16" # Should be #npcus can we set dynamically?
+    to: "openvino/model_server-build:2022.3-cuda" # We can specify a tag here right?
+  - dockerfile_path: DockerfileMakePackage
+    build_args:
+      - name: DLDT_PACKAGE_URL
+        value: ""
+      - name: BASE_OS
+        value: redhat
+      - name: NVIDIA
+        value: "1"
+      - name: BUILD_IMAGE
+        value: "openvino/model_server-build:2022.3-cuda"
+    inputs:
+      bin:
+        paths:
+        # - destination_dir: release_files/
+        #   source_path: /ovms.tar.gz # Don't think we actually need this one since the release file uses the .xz
+        - destination_dir: release_files/
+          source_path: /ovms.tar.xz
+    to: "openvino/model_server-pkg:2022.3-cuda"
+  - dockerfile_literal: |-
+      FROM ubi8/ubi
+      COPY release_files/ovms.tar.xz /
+      RUN yum -y install coreutils; sha256sum ovms.tar.gz > ovms.tar.gz.sha256
+    inputs:
+      bin:
+        paths:
+        - destination_dir: release_files/
+          source_path: /ovms.tar.xz.sha256
+    to: "shell-commands:2"
+  - dockerfile_path: Dockerfile.redhat
+    context_dir: release_files # Do we need to specify this to reset it? Need to move back to the root context dir here
+    # Need to copy the metadata file into .workspace/metadata.json, how?
+    build_args:
+      - name: INSTALL_RPMS_FROM_URL
+        value: ""
+      - name: GPU
+        value: "0" # This refers to the Intel GPU driver
+      - name: BASE_IMAGE
+        value: "docker.io/nvidia/cuda:11.8.0-runtime-ubi8"
+      - name: NVIDIA
+        value: "1"
+    to: "openvino/model_server-build:2022.3-cuda+cpu"
+  - dockerfile_path: Dockerfile.redhat
+    context_dir: release_files # Do we need to specify this to reset it? Need to move back to the root context dir here
+    # Need to copy the metadata file into .workspace/metadata.json, how?
+    build_args:
+      - name: INSTALL_RPMS_FROM_URL
+        value: ""
+      - name: GPU
+        value: "1" # This refers to the Intel GPU driver
+      - name: BASE_IMAGE
+        value: "docker.io/nvidia/cuda:11.8.0-runtime-ubi8"
+      - name: NVIDIA
+        value: "1"
+    to: "openvino/model_server-build:2022.3-cuda+intel-gpu"
+promotion: # Same as release image name? How would we determine the right image to push?
```

Comment:
We will push all of the images in the `red-hat-data-services` namespace in different imagestreams based on the image name with the tag `2022.3-gpu` in this case.

Reply:
Can we specify multiple tags?